src/components/image/image.component.tsx

```diff
@@ -2,19 +2,25 @@ import React from "react";
 import invariant from "invariant";
 import { StyledImage, StyledImageProps } from "./image.style";
 
-export const Image = ({ alt, src, children, ...rest }: StyledImageProps) => {
+export const Image = ({
+  alt,
+  decorative = false,
+  src,
+  children,
+  ...rest
+}: StyledImageProps) => {
   invariant(
     !src || !children,
     "The 'Image' component renders as an 'img' element when the 'src' prop is used and therefore does not accept children."
   );
 
   invariant(
-    !src || (alt && typeof alt === "string"),
-    "Please provide an 'alt' string when rendering the 'Image' component as an 'img' element."
+    !src || (alt && typeof alt === "string") || decorative,
```

Comment:
I realise you didn't add this, but why are we checking the `typeof` the `alt` prop as well when the interface says it's either `undefined | string`?

Reply:
Happy to approve for now as it's not breaking anything

Reply:
Cheers. I did look at this and think about changing it but getting the work done to make sure Nebula were not blocked took priority. I'll get something on the backlog so we don't forget to look at this in the near future 😄